common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -129,7 +130,7 @@ object CuratedRecipe {
       ),
       published = Some(
         ChangeRecord(
-          date = r.publicationDate.toInstant.toEpochMilli,
+          date = DateTime.now.toInstant.getMillis,
```

Comment:
I was thinking the the publication date of the atom should be `updated` when we publish a new version of the atom. The other option is to use `last-modified` but I am unsure if the difference is worth as we had on a legacy on article, what do you think?